### PR TITLE
add support for mprv

### PR DESCRIPTION
**Pull request by jwnhy**

### CODE REVIEW

File: src/register/mstatus.rs [hunk 1]
```diff
@@ -142,6 +142,12 @@ impl Mstatus {
         }
     }
 
+    /// Modify Memory PRiVilege
```
Comment (almindor) [inline src/register/mstatus.rs:145]: just a curious nit, why the odd capitalization of `PRiVilege` ?

Comment (almindor) [reply src/register/mstatus.rs:145]: oh nvm, I see that's how they call it in the risc-v docs, funky.

**Review by almindor — APPROVED** [review]
LGTM, thank you!

**Comment by Disasm** [comment]
bors cancel

**Comment by Disasm** [comment]
bors r=almindor

**Comment by Disasm** [comment]
@jwnhy could you reopen the PR?

**Comment by Disasm** [comment]
bors retry

**Comment by jwnhy** [comment]
@Disasm I have opened another PR for this.

**Pull request merged**